sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformationExec.scala

```diff
@@ -161,6 +162,25 @@ case class ScriptTransformationExec(
 
               if (scriptOutputReader != null) {
                 if (scriptOutputReader.next(scriptOutputWritable) <= 0) {
+                  // There can be a lag between reader read EOF and the process termination.
+                  // If the script fails to startup, this kind of error may be missed.
+                  // So explicitly waiting for the process termination.
+                  val timeout = SparkEnv.get.conf.get("spark.sql.transformation.exit.timeout",
+                    "1000").toInt
```

Comment:
We should use `SQLConf.get.getConf(SQLConf.TRANSFORMATION_EXIT_TIMEOUT)`.

Reply:
fixed.